pages/Identity/Overview.md

```diff
@@ -7,22 +7,222 @@ menu: Identity
 # Identity
 
 This specification is intended to cover the concept of identity within the protocol and how we represent it.
-It may include ideas for interfacing with external identity systems, methods to upgrade the identity system in the future and potential user authorization systems.
+It may include ideas for interfacing with external identity systems, methods to upgrade the identity system in the future and potential delegation systems.
 
 ## Specification Status
 
 | Version | Status |
 ---------- | ---------
-| 0.1     | Proposed |
+| 0.1     | Draft |
 
 ## Purpose
 
-TBD
+1. Provide the interface for a DSNP identity
 
 ## Assumptions
 
-TBD
+* Identity is on Ethereum for betanet
+* Identity will need to be moved from betanet to mainnet
 
 ## Details
 
-TBD
+Identity is comprised of several parts:
+
+* Ownership
+* Delegation
+* Identification
+* Discovery
+
+Additional constraints were placed upon any solution:
+
+* Pseudo Anonymity: A consistent identifier that defaults to disconnected from real world identity 
+* Flexibility: A interface that can produce multiple solutions
+* Extensibility: A default implementation that can be updated over time as DSNP changes
+
+## Ownership
+
+On the blockchain ownership is determined by the ability to sign messages.
+To this end, ownership of an identity is also determined by the address of the creator.
+(Options for allowing a delegate to create an identity on behalf of someone is described in the [Identity Factory](/Identity/Factory).)
+
+Creation of any new identity MUST be authorized by the owner's address.
+The official [Identity Factory](/Identity/Factory) must be used for initial creation of a new identity.
+
+Ownership is managed though [EIP-173](https://eips.ethereum.org/EIPS/eip-173) which provides methods that confirm ownership and facilitate ownership transfer.
+
+## Delegation
+
+Delegation allows adding public keys that are allowed to sign announcements
+or perform other actions in addition to the "owner" public key. 
+
+### Option 1: Without Permissions
+
+In this interface, the implementation determines permissions allowed for delegates.
```

Comment:
This description is the same as for option 2.  It would help to summarize the differences between the options.